hw/ip/otbn/code-snippets/README.md

```diff
@@ -0,0 +1,8 @@
+# OTBN code snippet collection
+
+This directory contains some code snippets that give examples of how
+to do various tasks in OTBN code.
+
+  - `otbn_bn_mul64.asm`: An example of how to use the 64-bit
```

Comment:
All other assembly files in OpenTitan end with `.S`. Should we keep this convention?

Reply:
Yeah, let's. I'd just copied the file name from Felix's repo, but let's switch over.